AxonDeepSeg/ads_utils.py

```diff
@@ -332,5 +333,25 @@ def get_file_extension(filename):
     extension = next((ext for ext in valid_extensions if str(filename).lower().endswith(ext)), None)
     return extension
 
+def check_available_gpus(gpu_id):
+    """ Get the number of available GPUs
+    Args:
+        gpu_id (int): Number representing the requested GPU ID for segmentation
+    Returns:
+        n_gpus: Number of available GPUs
+    """
+    from torch.cuda import device_count
```

Comment:
Not sure if the import will work once FSLeyes has already been loaded. We'll need to test this.

Reply:
I was able to test on rosenberg with X forwarding to my laptop. The display is very slow and by no means usable "in production" but good enough to test that sort of things.
The import seems to work, here is what I did:
1) Open fsleyes and install the plugin
2) Close and reopen fsleyes --> all 8 GPUs were available in settings
3) Close and reopen fsleyes with `CUDA_VISIBLE_DEVICES=0,1,2` --> 3 GPUs were available in settings

I put the import in the function as we usually do with torch in ivadomed. If you foresee any other issues with that for the plugin, we could also put the import at the top of the file. Let me know, thanks!

Reply:
Seems fine to me then 👍